Wartości Laboratorium EE v1.1.md

```diff
@@ -0,0 +1,167 @@
+## Wartości Laboratorium EE v1.1
```

Comment:
Czy to nie powinien być nagłówek najwyższego poziomu (`#` zamiast `##`)? Dlaczego v1.1? Czy jest gdzieś wersja 1.0?